R/run_stress_test.R

```diff
@@ -272,3 +293,11 @@ run_stress_test_impl <- function(asset_type_arg,
     )
   )
 }
+
+run_example <- function() {
```

Comment:
Avoid running the example in someone else's computer -- with potential for destroying valuable data.

It's so painful to have to remember those things and the time will come when I forget. It's best to follow CRAN's convention and not allow writing to  the user filespace at all. A package that violates those rules is hard to maintain.